drivers/pn532/pn532.c

```diff
@@ -181,14 +181,15 @@ static int _write(pn532_t *dev, char *buff, unsigned len)
     }
     else {
 #ifdef PN532_SUPPORT_SPI
-        spi_acquire(dev->conf->spi);
+        spi_acquire(dev->conf->spi, GPIO_UNDEF, SPI_MODE, SPI_CLK);
```

Comment:
Documentation in `spi.h` says cs shall be set to `SPI_CS_UNDEF` instead of `GPIO_UNDEF`